contracts/base/LoansBase.sol

```diff
@@ -626,4 +646,15 @@ contract LoansBase is LoansInterface, Base {
     function _createEscrow(uint256 loanID) internal returns (address) {
         return settings().escrowFactory().createEscrow(address(this), loanID);
     }
+
+    /**
+        @notice If the loan was tied to bank info (userId provided), reset the userId.
+        @param loanID loan id associated to the Escrow contract.
+     */
+    function _clearUserIdLoan(uint256 loanID) internal {
+        bytes32 userId = loans[loanID].userId;
+        if (userId != bytes32(0)) {
```

Comment:
Also we need to be checking that the `loanID` here is unsecured (`!loans[loanID].isSecured()`) before we clear the boolean